### PR TITLE
Start bridge with process compose

### PR DESCRIPTION
**Pull request by musitdev**
# Summary
- **RFCs**: [Link to RFC](#./link/to/rfc), [Link to RFC](#./link/to/rfc), or $\emptyset$.
- **Categories**: any of `protocol-units`, `networks`, `scripts`, `util`, `cicd`, or `misc`.

Add process compose start for the bridge.
# Changelog
Load config from suzuka node if present.
Add process compose script

# Testing

To start the bridge with suzuka node:
```
CELESTIA_LOG_LEVEL=FATAL CARGO_PROFILE=release CARGO_PROFILE_FLAGS=--release  nix develop --extra-experimental-features nix-command --extra-experimental-features flakes --command bash  -c "just bridge native build.setup.eth-local.celestia-local.bridge --keep-tui"
```
To start only the suzuka node without the bridge for test purpose:
```
CELESTIA_LOG_LEVEL=FATAL CARGO_PROFILE=release CARGO_PROFILE_FLAGS=--release  nix develop --extra-experimental-features nix-command --extra-experimental-features flakes --command bash  -c "just bridge native build.setup.eth-local.celestia-local --keep-tui"
```
To run the e2e bridge test:
```
CELESTIA_LOG_LEVEL=FATAL CARGO_PROFILE=release CARGO_PROFILE_FLAGS=--release  nix develop --extra-experimental-features nix-command --extra-experimental-features flakes --command bash  -c "just bridge native build.setup.eth-local.celestia-local.e2etest --keep-tui"
```
# Outstanding issues

### CODE REVIEW

File: protocol-units/bridge/setup/src/local.rs [hunk 1]
```diff
@@ -4,31 +4,12 @@ use aptos_sdk::types::LocalAccount;
...
-	tracing::info!("Bridge local setup");
-	//Eth init: Start anvil.
-	let anvil = setup_eth(&mut config.eth, &mut config.testing);
-	tracing::info!("Bridge after anvil");
-
-	//By default the setup deosn't start the Movement node.
-	Ok((config, anvil))
-}
-
 pub fn setup_eth(config: &mut EthConfig, testing_config: &mut TestingConfig) -> AnvilInstance {
```
Comment (andygolay) [inline protocol-units/bridge/setup/src/local.rs:13]: Do we still want this `local.rs` file, if `setup_eth` is the only `fn` in it?

Could `setup_eth` be moved somewhere else, perhaps into `deploy.rs`, and we remove `local.rs`?

**Review by 0xmovses — APPROVED** [review]
Thank you for this PR. We can successfully start up the full-suzuka-node with DA for local development with these process-compose scripts. 

I ran both the commands and they both work in the way you describe. Good stuff!

**Review by 0xmovses — CHANGES_REQUESTED** [review]
The integration tests `cargo test --test client_eth_movement -- --nocapture --test-threads=1` etc are failing now. Were you passing any additional ENV vars? 

I'll try removing `./movement` and re-running the client integration tests

**Comment by 0xmovses** [comment]
Ran the new command fore e2e test
```
CELESTIA_LOG_LEVEL=FATAL CARGO_PROFILE=release CARGO_PROFILE_FLAGS=--release  nix develop --extra-experimental-features nix-command --extra-experimental-features flakes --command bash  -c "just bridge native build.setup.eth-local.celestia-local.e2etest --keep-tui"
```
Sadly it just hangs and nothing starts, I have deleted the `./movement`, am I missing another step maybe?

**Comment by musitdev** [comment]
The integration client test failed with the LINKERROR that we had because of the native address wasn't set correctly. It's a Move issue. Perhaps @andygolay  has more EX on it.
Can you put the error you get when you run the command?

**Comment by 0xmovses** [comment]
```
2024-10-05T20:04:06.876235Z ERROR bridge_service::chains::movement::utils: Transaction submission error: Unknown error Transaction committed on chain, but failed execution: LINKER_ERROR
thread 'test_movement_client_should_successfully_call_lock_and_abort' panicked at protocol-units/bridge/integration-tests/tests/client_eth_movement.rs:199:14:
Failed to set timelock: CallError
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
FAILED
test test_movement_client_should_successfully_call_lock_and_complete

...
2024-10-05T20:04:49.186211Z  INFO bridge_service::chains::movement::utils: Starting send_aptos_transaction
2024-10-05T20:04:49.708474Z ERROR bridge_service::chains::movement::utils: Transaction submission error: Unknown error Transaction committed on chain, but failed execution: LINKER_ERROR
Error: No matching transaction found
FAILED

failures:

failures:
    test_movement_client_should_successfully_call_lock_and_abort
    test_movement_client_should_successfully_call_lock_and_complete

test result: FAILED. 8 passed; 2 failed; 2 ignored; 0 measured; 0 filtered out; finished in 181.26s
```
On main these pass.

File: protocol-units/bridge/setup/src/local.rs (hunk 1 above)
Comment (andygolay) [reply protocol-units/bridge/setup/src/local.rs:13]: Ah I wasn't looking at the whole file, I see what you mean now.

**Comment by 0xmovses** [comment]
I was able to test by starting the node with the bridge with 
```
CELESTIA_LOG_LEVEL=FATAL CARGO_PROFILE=release CARGO_PROFILE_FLAGS=--release  nix develop --extra-experimental-features nix-command --extra-experimental-features flakes --command bash  -c "just bridge native build.setup.eth-local.celestia-local.bridge --keep-tui"
```
And then running the integration tests in a separate terminal , `client_eth_movement` all pass
```
test result: ok. 10 passed; 0 failed; 2 ignored; 0 measured; 0 filtered out; finished in 187.25s
```

`client_movement_eth` one failure 
```
failures:
    test_movement_client_initiate_transfer
test result: FAILED. 2 passed; 1 failed; 1 ignored; 0 measured; 0 filtered out; finished in 259.84s
```

**Review by 0xmovses — APPROVED** [review]
Approving this as the one test that is failing is to be ignored as we discussed in the other PR.

**Pull request merged**